bnelson777/id.ly
# fix: Message Item not showing '>'

Test instructions:
	npm install
	react-native link
	react-native run-android
	or
	react-native run-ios

- Should be able to run react-native link without error
- Far right of message should be a '>'. not Kanji or a crossed out box
![linkissue](https://user-images.githubusercontent.com/22670901/39706028-ed395178-51c4-11e8-87f7-2d1c10fbae4c.png)
